main.py

```diff
@@ -25,8 +25,10 @@
     '%Y-%m-%d %H:%M:%S'
 )
 
-config_paths = ['/boot/zpui_config.json'] if not is_emulator else []
+config_paths = ['/boot/zpui_config.json', '/boot/pylci_config.json'] if not is_emulator else []
 config_paths.append(local_path('config.json'))
+#Using the .example config as a last resort
+config_paths.append(local_path('config.json.example'))
```

Comment:
since it's now actually used, it should'nt be called `example` anymore, but rather something like `config.default.json`